app/models/concerns/digital_object_concerns/publish_behavior.rb

```diff
@@ -99,13 +99,13 @@ def perform_publish_changes(publish_to: [], unpublish_from: [], publishing_user:
           end
 
           if new_highest_priority_doi_eligible_publish_target.nil?
-            Hyacinth::Config.external_identifier_adapter.tombstone(self.doi)
+            Hyacinth::Config.external_identifier_adapter.deactivate(self.doi)
           elsif (new_highest_priority_doi_eligible_publish_target != current_primary_doi_publish_target) || publish_to.include?(new_highest_priority_doi_eligible_publish_target)
             # We only need to perform an external identifier update if:
             # - We now have a new highest-priority doi publish target
             # or
             # - We are re-publishing to the existing highest-priority doi publish target.
-            Hyacinth::Config.external_identifier_adapter.update(self.doi, digital_object: self, target_url: self.citation_location)
+            Hyacinth::Config.external_identifier_adapter.update(self.doi, digital_object: self, target_url: self.citation_location, state: :active)
```

Comment:
Just leaving a note here (per our earlier conversation) about `state: :active` not being compatible with the current datacite adapter implementation.